portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,53 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,500,700|Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kristin Aoki</h1>
+      <img src="/images/last_page-9.png" alt="Image of Kristin Aoki" style="width:30%;display:block;margin-left:auto;
+        margin-right:auto;">
+      <p>Welcome and hello! I am software engineering student at Olin College of Engineering. When the weather is warm and
+          sunny you can find me hiking or on a golf course. When the weather isn't so nice, I will be found wrapped in a blanket
+          reading a book or watching a romantic comedy.
+      </p>
+      <p>I grew up in San Antonio, Texas and went to North East School of the Arts (NESA). At NESA I dedicated many hours to
+          the orchestra, playing the viola for many musicals and concerts. I decided to focus on fine arts in high school 
+          instead of tech/engineering because I knew it would be the last time I could really engage with this passion.
+      </p>
+      <p>I decided to go into the field of engineering because engineers have a solid hand in the future of the world. Without
+          engineering we would be a very rudimentary society. I want to use my knowledge to empower people and create a better
+          society. I am still unsure how I will accomplish this goal, but it is something I strive towards every day.
+      </p>
+      <h2>Projects</h2>
+      <h3>2018</h3>
+      <ul>
+          <li><a href="/photography.html">Photography (on going)</a></li>
+      </ul>
+      <h3>2019</h3>
+      <ul>
+          <li><a href="http://cw-guide.herokuapp.com/" target="blank">Candidate's Weekend Guide</a> - This web app was my 
+            final project for my class Software Design. I collaborate with two fellow classmates to design a web app for our
+            college, Olin College of Engineering.</li>
+          <li><a href="https://github.com/sd19spring/Death-Rate-Interactive-Map" target="blank">Interactive Map</a> - The 
+            interactive map shows the rates of death across the United States. </li>
+          <li><a href="https://popup.6seconds.org/">Six Seconds POP-UP Festival</a></li>
+          <li>Photography Book - </li>
+      </ul>
+      <h3>2020</h3>
+      <h2>Contact Me</h2>
+      <p>Email: kaoki@olin.edu</p>
```

Comment:
You can make this clickable with a mailto

So ...

`<p>Email: <a href="mailto:kaoki@olin.edu">kaoki@olin.edu</a></p>`

When users click it, it'll open a new email to you in their preferred email client.